activerecord/lib/active_record/fixtures.rb

```diff
@@ -456,6 +456,41 @@ class FixtureClassNotFound < ActiveRecord::ActiveRecordError # :nodoc:
   # In the above example, 'base' will be ignored when creating fixtures.
   # This can be used for common attributes inheriting.
   #
+  # == Composite Primary Key Fixtures
+  #
+  # Fixtures for composite primary key tables are fairly similar to normal tables.
+  # When using an id column, the column may be omitted as usual:
+  #
+  #   # app/models/book.rb
+  #   class Book < ApplicationRecord
+  #     self.primary_key = [:author_id, :id]
+  #     belongs_to :author, primary_key: :id
```

Comment:
Nit -- I'm wondering if we want to drop the `primary_key: :id` option now that Rails infers this automatically (https://github.com/rails/rails/pull/49113). Unless we think it adds more clarity to the example?
```suggestion
  #     belongs_to :author
```

Reply:
Oh TIL. Yeah let's do that